app/component/RoutePageControlPanel.js

```diff
@@ -467,7 +439,7 @@ class RoutePageControlPanel extends React.Component {
             <FormattedMessage id="route-guide" defaultMessage="Route guide" />
           </h1>
         </div>
-        {isOnDemandTaxi && <CallAgencyWarning route={route} />}
+        {route.type === 715 && <CallAgencyWarning route={route} />}
```

Comment:
I think we should keep `isOnDemandTaxi` here to point out when this warning is displayed.

`const isOnDemandTaxi = route.type === 715;`